models/brats_mri_generative_diffusion/docs/README.md

```diff
@@ -0,0 +1,168 @@
+# Model Overview
+A pre-trained model for volumetric (3D) Brats MRI 3D Latent Diffusion Generative Model.
+
+This model is trained on BraTS 2016 and 2017 data from [Medical Decathlon](http://medicaldecathlon.com/), using the Latent diffusion model [1].
+
+![model workflow](https://developer.download.nvidia.com/assets/Clara/Images/monai_brain_image_gen_ldm3d_network.png)
+
+This model is a generator for creating images like the Flair MRIs based on BraTS 2016 and 2017 data. It was trained as a 3d latent diffusion model and accepts Gaussian random noise as inputs to produce an image output. The `train_autoencoder.json` file describes the training process of the variational autoencoder with GAN loss. The `train_diffusion.json` file describes the training process of the 3D latent diffusion model.
+
+**This is a demonstration network meant to just show the training process for this sort of network with MONAI. To achieve better performance, users need to use larger dataset like [Brats 2021](https://www.synapse.org/#!Synapse:syn25829067/wiki/610865) and have GPU with memory larger than 32G to enable larger networks and attention layers.**
+
+#### Example synthetic image
+![Example synthetic image](https://developer.download.nvidia.com/assets/Clara/Images/monai_brain_image_gen_ldm3d_example_generation.png)
```

Comment:
Hi @yiheng-wang-nv, that's probably a better way to integrate it. We may be able to add this to the template as well so that as we update models in the future, if it makes sense, to have a visual in the overview section.

Reply:
Yes, I added a sentence